src/rnp/rnpcfg.c

```diff
@@ -426,6 +426,9 @@ rnp_path_compose(const char *dir, const char *subdir, const char *filename, char
     }
 
     /* concatenating dir, subdir and filename */
+    if (strlen(dir) > MAXPATHLEN - 1)
+        return false;
```

Comment:
Can you add braces to these one-line conditionals? ([Dev Guide](https://github.com/riboseinc/rnp/blob/master/Developers%20Guide.md#general-guidelines))